server/db/seed.js

```diff
@@ -0,0 +1,30 @@
+const faker = require('faker');
+const db = require('./index.js');
+
+
+const carInfo = () => {
+  for (let i = 0; i < 65000; i += 1) {
+    db.CarInfo.create({
+      id: i + 1,
```

Comment:
you shouldn't ever specify the ID.. you should let the database handle it

Reply:
I was actually doing that just to see and accurate number of what number it would stop inserting at but I'll edit that out!